Tests/Acceptance/inc/locators.json

```
@@ -60,6 +60,14 @@
       "submitFinalize": "sbtnSignSingle",
       "submitConfirm": "sbtnOk",
       "backToShop": "//input[@name='back2Shop']"
+    },
+    "payolutionInv":{
+      "dateOfBirth": "//input[@id='dateOfBirthpayolution-inv']",
+      "termsCheckbox": "//input[@name='terms_checkbox']"
+    },
+    "payolutionB2B":{
+      "companyName": "//input[@id='wdCompanyName']",
```

Locators with IDs can also be specified directly, e.g.:
```suggestion
      "companyName": "wdCompanyName",
```

---

👍